custom.sample.php

```diff
@@ -15,16 +16,206 @@
-	</hr>
-	<?php print_r($ant->get('paymentHistory')); ?>
-</pre>
+<html lang="en">
+  <head>
+    <title>Our Mining Stats</title>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
+  </head>
+  <body>
+  <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+  <a class="navbar-brand" href="#">Navbar</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
+  <div class="collapse navbar-collapse" id="navbarSupportedContent">
+    <ul class="navbar-nav mr-auto">
+      <li class="nav-item active">
+        <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+      </li>
+  </ul>
+   </div>
+  </nav>
+  
+  <table class="table table-dark">
+  <thead>
+  	<tr>
+    <th colspan="4"><center>Account info</center></th>
+    </tr>
+    <tr>
+      <th scope="col">Balance</th>
+      <th scope="col">24 hour earnings</th>
+      <th scope="col">Est. Weekly Earnings</th>
+     <th scope="col">Est. Monthly Earnings</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><?php print "$balance LTC"; ?></td>
+      <td><?php print "$daily LTC"; ?></td>
+      <td><?php print $daily*7; print "LTC"; ?></td>
+      <td><?php print $daily*30; print "LTC"; ?></td>
+    </tr>
+    <tr>
+      <td><?php print "$";print round($balanceUSD,2); ?></td>
+      <td><?php print "$"; print round($daily*$usd,2); ?></td>
+      <td><?php print "$"; print round(($daily*7)*$usd,2); ?></td>
+      <td><?php print "$"; print round(($daily*30)*$usd,2); ?></td>
+    </tr>
+  </tbody>
+</table>
+  
+  <br><hr><br>
+  
+    <table class="table table-dark">
+  <thead>
+  	<tr>
+    <th colspan="4"><center>Hashrate</center></th>
+    </tr>
+    <tr>
+      <th scope="col">Hashrate 10 mins</th>
+      <th scope="col">Hashrate 30 mins</th>
+      <th scope="col">Hashrate 1 hour</th>
+     <th scope="col">Hashrate 1 day</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><?php print round(($last10m/1000),3); print " GH/s"; ?></td>
+      <td><?php print round(($last30m/1000),3); print " GH/s"; ?></td>
+      <td><?php print round(($last1h/1000),3); print " GH/s"; ?></td>
+      <td><?php print round(($last1d/1000),3); print " GH/s"; ?></td>
+    </tr>
+  </tbody>
+</table>
+  
+    <br><hr><br>
+  
+    <table class="table table-dark">
+  <thead>
+  	<tr>
+    <th colspan="4"><center>Last 10 Payments</center></th>
+    </tr>
+    <tr>
+      <th scope="col">Time & Date</th>
```

Comment:
& Need to be escaped

Reply:
It stands for "and" in English

Reply:
Yeah - i think this is the meaning in the whole world (Chinese not i think so :_D)
"&" is an Special escape charactere in html.

If you will print a "&" you should use the string "&amp;"
https://dev.w3.org/html5/html-author/charref